### ISSUE TITLE
Bad image path in SASS files

### ISSUE THREAD

**Issue by santypr** [madskristensen/WebEssentials2013#1720]
For the last weeks I'm experiencing a big problem when I write an image url in a SASS file. The compiler y changing the path to an absolute path so the system cant't find the image.

Original in SASS file
  background-image: url(../Images/Icons/search-magenta-32.png);

Compiled CSS file
  background-image: url(D:\Development\myexample\mvc\Content\Images\search-magenta-32.png);

I've tested using '' and without them

**Comment by am11** [comment]
1. Which version of WE (or node-sass to be precise if you can dig it up**) is that?
2. Do you get the same results if Autoprefixer and RTLCss are disabled under Tools > Options > Web Essentials > CSS?


<I>** There is a nodejs executable, where WE is installed: `%LocalAppData%\Microsoft\VisualStudio\12.0\Extensions\<some-unique-hash-code>\Resources\nodejs\node.exe`. Open cmd or PowerShell, enter this directory using `cd` and run this command: `node tools\node_modules\node-sass\bin\node-sass --version`.</I>

**Comment by santypr** [comment]
It doesn't work. If I use relative path the compiler converts the path to an absolute system path

**Comment by am11** [comment]
You didn't answer the questions though..

**Comment by santypr** [comment]
My version is 3.5 and I updated yesterday with the nightly build to 3.5.1 and the result is the same with Autoprefixer and RTLCSS disabled :(

**Comment by am11** [comment]
You project is `D:` drive. That is probably not the system drive. If that is the case, do you get the same result when the project is in system (`C:`) drive?

**Comment by santypr** [comment]
D: is my secondary HD. The problem persists if I change the project to C:

**Comment by am11** [comment]
Wait, are you using WE2013 with VS2013? :) 

Because WE2013's latest version is 2.5.3.

**Comment by santypr** [comment]
That's right, I write the numbers in wrong order. My version is 2.5.3.1 as you can see in the screenshot
![we](https://cloud.githubusercontent.com/assets/7237337/5798816/42208d1c-9fc3-11e4-9fa2-2e325f0c2dab.PNG)

**Comment by lepipele** [comment]
+1 Same problem here
